bin/mlive

```diff
@@ -301,44 +382,103 @@ LivenessTester.prototype.start = function ()
 	fprintf(stderr, 'will test for liveness: %s\n', args.ml_reads ?
 	    (args.ml_writes ? 'reads, writes' : 'reads') : 'writes');
 	fprintf(stderr, 'assuming %d metadata shards\n', args.ml_nshards);
-	fprintf(stderr, 'using base paths: %s\n', args.ml_paths.join(', '));
+	fprintf(stderr, 'testing: %s\n', args.ml_test_buckets ? 'buckets' :
+	    'directory manta');
+	if (args.ml_test_buckets) {
+		fprintf(stderr, 'using buckets: %s\n',
+		    args.ml_buckets.join(', '));
+	} else {
+		fprintf(stderr, 'using base paths: %s\n',
+		    args.ml_paths.join(', '));
+	}
 	fprintf(stderr, 'time between requests: %d ms\n', args.ml_delay);
 	fprintf(stderr, 'maximum outstanding requests: %d\n', args.ml_maxrqs);
 	fprintf(stderr, 'environment:\n');
 	for (key in process.env) {
-		if (key.substr(0, 'MANTA_'.length) != 'MANTA_')
+		if (key.substr(0, 'MANTA_'.length) !== 'MANTA_')
 			continue;
 
 		fprintf(stderr, '    %7s = %s\n', key, process.env[key]);
 	}
 
+	var multiplier = 15;
 	this.mlt_requests = [];
-	args.ml_paths.forEach(function (basepath) {
-		var paths;
 
-		paths = makePathList(basepath, args.ml_nshards);
-		paths.forEach(function (p) {
+	/*
```

Comment:
This comment could do with being updated. In buckets land we shard based on the object name, so it would be good to make sure that's reflected here. I think the rest of it makes sense though to make sure all shards are covered but I'm not 100%.

Reply:
Sounds good, will do.